lake/api/manage/config.go

```diff
@@ -19,32 +19,27 @@ func (c *Config) poolConfig(p *pools.Config) PoolConfig {
 		if p.Name != pconf.Pool && p.ID.String() != pconf.Pool {
 			continue
 		}
-		if pconf.Interval == nil {
-			pconf.Interval = c.Interval
-		}
 		if pconf.Branch == "" {
 			pconf.Branch = "main"
 		}
 		return pconf
 	}
 	return PoolConfig{
-		Pool:     p.Name,
-		Branch:   "main",
-		Interval: c.Interval,
-		Vectors:  c.Vectors,
+		Pool:    p.Name,
+		Branch:  "main",
+		Vectors: c.Vectors,
 	}
 }
 
-type PoolConfig struct {
-	Pool     string         `yaml:"pool"`
-	Branch   string         `yaml:"branch"`
-	Interval *time.Duration `yaml:"interval"`
```

Comment:
Why remove this field?

Reply:
It just seemed like having a per-pool interval added a lot of complexity to the code that might not ultimately get used so I'm simplified things and just put interval on a global level. Do you think this is a mistake?

Reply:
I imagine the real version of manage (not this toy version we currently have) will not have intervals but rather will compact opportunistically while data comes in. Following this logic it made sense to me to just keep this as simple as possible.

Reply:
Makes sense to me.